R/calculate-effect-sizes.R

```diff
@@ -460,9 +526,61 @@ batch_calc_ES <- function(dat,
           .groups = "drop"
         )
       
+    } else if (weighting %in% c("nA", "n_A")) {
+      ES_weights <- 
+        ES_ests_long %>% 
+        dplyr::left_join(n_weights)
+      
+      res <- 

```

Comment:
Can these conditions all be handled by using a variable called `weights`? I think this would simplify all the conditional arguments.